src/DataProvider/Product/SearchCriteriaBuilder.php

```diff
@@ -0,0 +1,250 @@
+<?php
+/**
+ * @category    ScandiPWA
+ * @package     ScandiPWA_CatalogGraphQl
+ * @author      Denis Protassoff <info@scandiweb.com>
+ * @copyright   Copyright (c) 2022 Scandiweb, Ltd (https://scandiweb.com)
+ */
+
+declare(strict_types=1);
+
+namespace ScandiPWA\CatalogGraphQl\DataProvider\Product;
+
+use Magento\Catalog\Api\Data\EavAttributeInterface;
+use Magento\Framework\Api\FilterBuilder;
+use Magento\Framework\Api\Search\FilterGroupBuilder;
+use Magento\Framework\Api\Search\SearchCriteriaInterface;
+use Magento\Framework\Api\SortOrder;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\GraphQl\Query\Resolver\Argument\SearchCriteria\Builder;
+use Magento\Catalog\Model\Product\Visibility;
+use Magento\Framework\Api\SortOrderBuilder;
+use Magento\CatalogGraphQl\DataProvider\Product\SearchCriteriaBuilder as MagentoSearchCriteriaBuilder;
+
+/**
+ * Class SearchCriteriaBuilder
+ *
+ * @package ScandiPWA\CatalogGraphQl\DataProvider\Product
+ */
+class SearchCriteriaBuilder extends MagentoSearchCriteriaBuilder
+{
+    /**
+     * @var ScopeConfigInterface
+     */
+    private $scopeConfig;
```

Comment:
Make all properties as protected. Otherwise on extending class it will be required to define properties again.